docs/infrastructure/native-image.md

```diff
@@ -117,7 +117,20 @@ It is possible that different classes are reflectively accessed on different
 platforms. In that case it may be necessary to run the agent on multiple
 platforms and merge the configs. If the conflicts were conflicting (i.e. some
 reflectively accessed classes existed only on one platform), it may be necessary
-to maintain separate configs for each platform. Currently in the Launcher this
-is not the case - the reflective accesses seem to be platform independent, as
-the launcher built with a config created on Linux runs successfully on other
-platforms.
+to maintain separate configs for each platform. Currently, the
+`native-image-agent` is not available on Windows, so Windows-specific reflective
+accesses may have to be gathered manually. For some types of accesses it may be
+possible to force the Windows-specific code paths to run on Linux and gather
+these accesses semi-automatically. That is currently the case with
+archive-related accesses - while the Linux launcher never unpacks ZIP files, we
+can manually force it to do so, to register the reflection configuration that
+will than be used on Windows to enable ZIP extraction.
```

Comment:
Uh that's a very good question indicating that it should be documented, because honestly - I forgot how I did that.

The simplest way (and I think that is exactly what I did), is to add the following code to the `main` method:

```
Archive.extractArchive(Path.of("enso-engine-windows.zip"), Path.of("somewhere"), None)
```

and run it.

Another way to do this, is force the Linux launcher to think it is running on Windows by defining `ENSO_OPERATING_SYSTEM=windows` and running `install engine` which will install a Windows engine version. But this will only work inside of a portable distribution (to avoid a failed attempt at looking up `%LocalAppData%` and other Windows-specific stuff).

I think adding this line of code to main is the most robust solution. If you think it is better, we can add an `--internal-archive-test` flag that will extract some archive to trigger this without modifying sources.

Reply:
Please document this process.